### ISSUE TITLE
Is there possible to insert some new class while not forget origin class.

### ISSUE THREAD

**Issue by trarynight** [bytedance/fc-clip#5]
Thanks for your great work. when i training for my own datasets, i found the trained model forget origin class. Is there any process to avoid this. I mean the new model can preserve origin class recognition ability or drop not so much? 
Appreciate for your reply.

**Comment by cornettoyu** [comment]
Hi,

Thanks for your interest in our work. If you mean fine-tuning with your own data on top of the coco-pretrained checkpoint, you can add COCO's metadata to your own dataset. If you mean the fine-tuned model performs bad on the coco classes after fine-tuning, you may try lower learning rate/training length, jointly train with COCO, etc.

Best,

**Comment by trarynight** [comment]
Thank you for your quick response.
